src/protocol_versions.json

```diff
@@ -1,22 +1,26 @@
 {
-    "1.0.0": {
-        "min": "0.1.0",
-        "max": "0.1.0"
-    },
-    "1.0.1": {
-        "min": "0.1.0",
-        "max": "0.1.0"
-    },
-    "1.0.2": {
-        "min": "0.1.0",
-        "max": "0.1.0"
-    },
-    "1.0.3": {
-        "min": "0.1.0",
-        "max": "0.1.0"
-    },
-    "1.0.4": {
-        "min": "0.1.0",
-        "max": "0.1.0"
-    }
+  "1.0.0": {
+    "min": "0.1.0",
+    "max": "0.1.0"
+  },
+  "1.0.1": {
+    "min": "0.1.0",
+    "max": "0.1.0"
+  },
+  "1.0.2": {
+    "min": "0.1.0",
+    "max": "0.1.0"
+  },
+  "1.0.3": {
+    "min": "0.1.0",
+    "max": "0.1.0"
+  },
+  "1.0.4": {
+    "min": "0.1.0",
+    "max": "0.1.0"
+  },
+  "1.0.5": {
+    "min": "0.3.1",
```

Comment:
'0.3.1' version has backward compatibility, so it seems that it can support up to 0.1.0(min: 0.1.0). What do you think?

Reply:
Actually, we have released some blockchain clients without changing the package version and some of them may not be compatible with v0.3.1.. I think it would be better/less trouble if we up the min version now.

Reply:
Time to say goodbye to 0.1.0... OK. It's a good idea :+1: